solutions/usaco-615.mdx

```diff
@@ -2,16 +2,53 @@
 id: usaco-615
 source: USACO Bronze 2016 February
 title: Milk Pails
-author: Danh Ta Chi Thanh
+author: Ryan Chou (Python), Danh Ta Chi Thanh (C++)
 ---
 
 [Official Analysis (Java)](http://www.usaco.org/current/data/sol_pails_bronze_feb16.html)
 
 <LanguageSection>
 
+## Implementation
+	
```

Comment:
this should go outside the LanguageSection tag

Reply:
got it, fixing it now.

Reply:
I've moved it. Thanks for the quick reply!